### PR TITLE
drm: apply drm to dash outputs

### PR DESCRIPTION
**Pull request by zsiec**
```
> GET /encodes/dash/one_off/CBS/manifest.mpd?ContentId=FPSLiveTest&keyId=11111111-1111-1111-1111-a21ec817c1e9&ls_session=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjNkNjg4NGJmLWViMDktNDA1Zi1hOWZjLWU0NGE1NmY3NjZiNiJ9.eyJzdWIiOiJVU18xNTAwMDU2MTkiLCJpc3MiOiJjYnMiLCJlbnQiOlt7ImJpZCI6IkFsbEFjY2Vzc01haW4iLCJlcGlkIjo4fV0sImp0aSI6ImM0MjE5ZjllLTI2YTMtNGE0OC1hMDY4LTk2NmVmMWY1MGI3NyIsImV4cCI6MTY2MDQyMzA0NjAwMCwiaWF0IjoxNTY1NzI3MTkxMTE1LCJhaWQiOiJjYnNpIn0.x0pK86BjXrtfg0CkQ3VH8_co6567wNiRBwpgMhSHH1s HTTP/1.1
> Host: localhost:1999
> User-Agent: curl/7.54.0
> Accept: */*
>
< HTTP/1.1 200 OK
< Server: nginx/1.16.1
< Date: Fri, 20 Sep 2019 19:51:49 GMT
< Content-Type: application/dash+xml
< Content-Length: 6873
< Connection: keep-alive
< Access-Control-Allow-Headers: *
< Access-Control-Expose-Headers: Server,range,Content-Length,Content-Range
< Access-Control-Allow-Methods: GET, HEAD, OPTIONS
< Access-Control-Allow-Origin: *
< Accept-Ranges: bytes
<
<?xml version="1.0"?>
<MPD
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns="urn:mpeg:dash:schema:mpd:2011"
    xsi:schemaLocation="urn:mpeg:dash:schema:mpd:2011 http://standards.iso.org/ittf/PubliclyAvailableStandards/MPEG-DASH_schema_files/DASH-MPD.xsd"
    type="static"
    mediaPresentationDuration="PT30.421S"
    minBufferTime="PT4S"
    profiles="urn:mpeg:dash:profile:isoff-main:2011">
  <Period>
    <AdaptationSet
        id="1"
        segmentAlignment="true"
        maxWidth="626"
        maxHeight="360"
        maxFrameRate="45000/1877">
      <Representation
          id="v1-x3"
          mimeType="video/mp4"
          codecs="avc1.640028"
          width="626"
          height="360"
          frameRate="45000/1877"
          sar="1:1"
          startWithSAP="1"
          bandwidth="381863">
        <SegmentList timescale="1000" duration="4000" startNumber="1">
          <Initialization sourceURL="init-v1-x3.mp4"/>
          <SegmentURL media="segment-1-v1-x3.m4s"/>
          <SegmentURL media="segment-2-v1-x3.m4s"/>
          <SegmentURL media="segment-3-v1-x3.m4s"/>
          <SegmentURL media="segment-4-v1-x3.m4s"/>
          <SegmentURL media="segment-5-v1-x3.m4s"/>
          <SegmentURL media="segment-6-v1-x3.m4s"/>
          <SegmentURL media="segment-7-v1-x3.m4s"/>
          <SegmentURL media="segment-8-v1-x3.m4s"/>
        </SegmentList>
        <ContentProtection schemeIdUri="urn:mpeg:dash:mp4protection:2011" value="cenc"/>
        <ContentProtection xmlns:mspr="urn:microsoft:playready" schemeIdUri="urn:uuid:9a04f079-9840-4286-ab92-e65be0885f95">
          <mspr:pro>PGNlbmM6cHNzaCB4bWxuczpjZW5jPSJ1cm46bXBlZzpjZW5jOjIwMTMiPkFBQURCSEJ6YzJnQUFBQUFtZ1R3ZVpoQVFvYXJrdVpiNEloZmxRQUFBdVRrQWdBQUFRQUJBTm9DUEFCWEFGSUFUUUJJQUVVQVFRQkVBRVVBVWdBZ0FIZ0FiUUJzQUc0QWN3QTlBQ0lBYUFCMEFIUUFjQUE2QUM4QUx3QnpBR01BYUFCbEFHMEFZUUJ6QUM0QWJRQnBBR01BY2dCdkFITUFid0JtQUhRQUxnQmpBRzhBYlFBdkFFUUFVZ0JOQUM4QU1nQXdBREFBTndBdkFEQUFNd0F2QUZBQWJBQmhBSGtBVWdCbEFHRUFaQUI1QUVnQVpRQmhBR1FBWlFCeUFDSUFJQUIyQUdVQWNnQnpBR2tBYndCdUFEMEFJZ0EwQUM0QU1BQXVBREFBTGdBd0FDSUFQZ0E4QUVRQVFRQlVBRUVBUGdBOEFGQUFVZ0JQQUZRQVJRQkRBRlFBU1FCT0FFWUFUd0ErQUR3QVN3QkZBRmtBVEFCRkFFNEFQZ0F4QURZQVBBQXZBRXNBUlFCWkFFd0FSUUJPQUQ0QVBBQkJBRXdBUndCSkFFUUFQZ0JCQUVVQVV3QkRBRlFBVWdBOEFDOEFRUUJNQUVjQVNRQkVBRDRBUEFBdkFGQUFVZ0JQQUZRQVJRQkRBRlFBU1FCT0FFWUFUd0ErQUR3QVN3QkpBRVFBUGdCRkFGSUFSUUJTQUVVQVVnQkZBRklBUlFCU0FFVUFVZ0JGQUdFQVNRQmxBSGtBUWdCbUFFSUFOZ0JSQUQwQVBRQThBQzhBU3dCSkFFUUFQZ0E4QUVNQVNBQkZBRU1BU3dCVEFGVUFUUUErQUVrQVZRQkVBRlVBVFFCV0FFNEFSUUJvQUZFQVZRQTlBRHdBTHdCREFFZ0FSUUJEQUVzQVV3QlZBRTBBUGdBOEFFd0FRUUJmQUZVQVVnQk1BRDRBYUFCMEFIUUFjQUJ6QURvQUx3QXZBSElBYmdCbkFDNEFjd0IwQUdFQVp3QmxBQzRBYndCMEFIUUFMZ0JwQUhJQVpBQmxBSFFBYndBdUFHTUFid0J0QUM4QWJBQnBBR01BWlFCdUFITUFaUUJUQUdVQWNnQjJBR1VBY2dBdkFIQUFiQUJoQUhrQWNnQmxBR0VBWkFCNUFDOEFkZ0F4QUM4QVl3QmlBSE1BYVFBdkFHd0FhUUJqQUdVQWJnQnpBR1VBUHdCakFHOEFiZ0IwQUdVQWJnQjBBRWtBWkFBOUFFWUFVQUJUQUV3QWFRQjJBR1VBVkFCbEFITUFkQUE4QUM4QVRBQkJBRjhBVlFCU0FFd0FQZ0E4QUM4QVJBQkJBRlFBUVFBK0FEd0FMd0JYQUZJQVRRQklBRVVBUVFCRUFFVUFVZ0ErQUE9PTwvY2VuYzpwc3NoPg==</mspr:pro>
        </ContentProtection>
        <ContentProtection xmlns:cenc="urn:mpeg:cenc:2013" schemeIdUri="urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed" cenc:default_KID="11111111-1111-1111-1111-a21ec817c1e9">
          <cenc:pssh>AAAAr3Bzc2gAAAAA7e+LqXnWSs6jyCfc1R0h7QAAAI88Y2VuYzpwc3NoIHhtbG5zOmNlbmM9InVybjptcGVnOmNlbmM6MjAxMyI+QUFBQVFYQnpjMmdBQUFBQTdlK0xxWG5XU3M2anlDZmMxUjBoN1FBQUFDRUlBUklRRVJFUkVSRVJFUkVSRWFJZXlCZkI2U0lMUmxCVFRHbDJaVlJsYzNRPTwvY2VuYzpwc3NoPg==</cenc:pssh>
        </ContentProtection>
      </Representation>
    </AdaptationSet>
    <AdaptationSet
        id="2"
        segmentAlignment="true">
      <AudioChannelConfiguration
          schemeIdUri="urn:mpeg:dash:23003:3:audio_channel_configuration:2011"
          value="1"/>
      <Representation
          id="a1-x3"
          mimeType="audio/mp4"
          codecs="mp4a.40.2"
          audioSamplingRate="48000"
          startWithSAP="1"
          bandwidth="128146">
        <SegmentList timescale="1000" duration="4000" startNumber="1">
          <Initialization sourceURL="init-a1-x3.mp4"/>
          <SegmentURL media="segment-1-a1-x3.m4s"/>
          <SegmentURL media="segment-2-a1-x3.m4s"/>
          <SegmentURL media="segment-3-a1-x3.m4s"/>
          <SegmentURL media="segment-4-a1-x3.m4s"/>
          <SegmentURL media="segment-5-a1-x3.m4s"/>
          <SegmentURL media="segment-6-a1-x3.m4s"/>
          <SegmentURL media="segment-7-a1-x3.m4s"/>
          <SegmentURL media="segment-8-a1-x3.m4s"/>
        </SegmentList>
        <ContentProtection schemeIdUri="urn:mpeg:dash:mp4protection:2011" value="cenc"/>
        <ContentProtection xmlns:mspr="urn:microsoft:playready" schemeIdUri="urn:uuid:9a04f079-9840-4286-ab92-e65be0885f95">
          <mspr:pro>PGNlbmM6cHNzaCB4bWxuczpjZW5jPSJ1cm46bXBlZzpjZW5jOjIwMTMiPkFBQURCSEJ6YzJnQUFBQUFtZ1R3ZVpoQVFvYXJrdVpiNEloZmxRQUFBdVRrQWdBQUFRQUJBTm9DUEFCWEFGSUFUUUJJQUVVQVFRQkVBRVVBVWdBZ0FIZ0FiUUJzQUc0QWN3QTlBQ0lBYUFCMEFIUUFjQUE2QUM4QUx3QnpBR01BYUFCbEFHMEFZUUJ6QUM0QWJRQnBBR01BY2dCdkFITUFid0JtQUhRQUxnQmpBRzhBYlFBdkFFUUFVZ0JOQUM4QU1nQXdBREFBTndBdkFEQUFNd0F2QUZBQWJBQmhBSGtBVWdCbEFHRUFaQUI1QUVnQVpRQmhBR1FBWlFCeUFDSUFJQUIyQUdVQWNnQnpBR2tBYndCdUFEMEFJZ0EwQUM0QU1BQXVBREFBTGdBd0FDSUFQZ0E4QUVRQVFRQlVBRUVBUGdBOEFGQUFVZ0JQQUZRQVJRQkRBRlFBU1FCT0FFWUFUd0ErQUR3QVN3QkZBRmtBVEFCRkFFNEFQZ0F4QURZQVBBQXZBRXNBUlFCWkFFd0FSUUJPQUQ0QVBBQkJBRXdBUndCSkFFUUFQZ0JCQUVVQVV3QkRBRlFBVWdBOEFDOEFRUUJNQUVjQVNRQkVBRDRBUEFBdkFGQUFVZ0JQQUZRQVJRQkRBRlFBU1FCT0FFWUFUd0ErQUR3QVN3QkpBRVFBUGdCRkFGSUFSUUJTQUVVQVVnQkZBRklBUlFCU0FFVUFVZ0JGQUdFQVNRQmxBSGtBUWdCbUFFSUFOZ0JSQUQwQVBRQThBQzhBU3dCSkFFUUFQZ0E4QUVNQVNBQkZBRU1BU3dCVEFGVUFUUUErQUVrQVZRQkVBRlVBVFFCV0FFNEFSUUJvQUZFQVZRQTlBRHdBTHdCREFFZ0FSUUJEQUVzQVV3QlZBRTBBUGdBOEFFd0FRUUJmQUZVQVVnQk1BRDRBYUFCMEFIUUFjQUJ6QURvQUx3QXZBSElBYmdCbkFDNEFjd0IwQUdFQVp3QmxBQzRBYndCMEFIUUFMZ0JwQUhJQVpBQmxBSFFBYndBdUFHTUFid0J0QUM4QWJBQnBBR01BWlFCdUFITUFaUUJUQUdVQWNnQjJBR1VBY2dBdkFIQUFiQUJoQUhrQWNnQmxBR0VBWkFCNUFDOEFkZ0F4QUM4QVl3QmlBSE1BYVFBdkFHd0FhUUJqQUdVQWJnQnpBR1VBUHdCakFHOEFiZ0IwQUdVQWJnQjBBRWtBWkFBOUFFWUFVQUJUQUV3QWFRQjJBR1VBVkFCbEFITUFkQUE4QUM4QVRBQkJBRjhBVlFCU0FFd0FQZ0E4QUM4QVJBQkJBRlFBUVFBK0FEd0FMd0JYQUZJQVRRQklBRVVBUVFCRUFFVUFVZ0ErQUE9PTwvY2VuYzpwc3NoPg==</mspr:pro>
        </ContentProtection>
        <ContentProtection xmlns:cenc="urn:mpeg:cenc:2013" schemeIdUri="urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed" cenc:default_KID="11111111-1111-1111-1111-a21ec817c1e9">
          <cenc:pssh>AAAAr3Bzc2gAAAAA7e+LqXnWSs6jyCfc1R0h7QAAAI88Y2VuYzpwc3NoIHhtbG5zOmNlbmM9InVybjptcGVnOmNlbmM6MjAxMyI+QUFBQVFYQnpjMmdBQUFBQTdlK0xxWG5XU3M2anlDZmMxUjBoN1FBQUFDRUlBUklRRVJFUkVSRVJFUkVSRWFJZXlCZkI2U0lMUmxCVFRHbDJaVlJsYzNRPTwvY2VuYzpwc3NoPg==</cenc:pssh>
        </ContentProtection>
      </Representation>
    </AdaptationSet>
  </Period>
</MPD>```

### CODE REVIEW

File: drm-proxy/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ FROM golang:1.13-alpine3.10 AS build
 RUN apk add --no-cache ca-certificates curl git build-base
 
 RUN mkdir /drm-proxy
-RUN curl -sL https://d1jfzjx68gj8xs.cloudfront.net/items/2g3G2Y0y3L2f2X2U3p1b/cbsinteractive-drm-proxy-51bef0d482b6ade0ce6aa527003033b6db69e996.tar.gz | tar -C /drm-proxy --strip 1 -xz
+RUN curl -sL https://github.com/zsiec/drm-proxy/tarball/master | tar -C /drm-proxy --strip 1 -xz
```
Comment (zsiec) [inline drm-proxy/Dockerfile]: @flavioribeiro I put this on my account pending permission to open-source this on `cbsinteractive`. Do you think we can make our `cbsinteractive` version public or should we re-work this to enable it to work with private repos?

Comment (flavioribeiro) [reply drm-proxy/Dockerfile]: @zsiec I don't see any issues in bringing the current implementation to our @cbsinteractive account

File: otfp/nginx.conf [hunk 2]
```diff
@@ -146,6 +148,19 @@ http {
         location /encodes/dash {
```
Comment (flavioribeiro) [inline otfp/nginx.conf]: for testing purposes, do you think we should have a `/encodes/hls-drm` and `/encodes/hls` in the default config?

Comment (zsiec) [reply otfp/nginx.conf]: absolutely, I've added this

**Pull request merged**